tensorflow/python/keras/distribute/distribute_strategy_test.py

```diff
@@ -1604,6 +1604,8 @@ def test_regularizer_loss(self, distribution):
       self.assertEqual(-1.0, v)
 
 
+@testing_utils.run_all_without_tensor_float_32(
+    'Uses Dense layers, which call matmul')
```

Comment:
This seems scary -- ideally we should be able to run dense layers in TF32 without noticing right?

How much would we have to adjust the tolerances here to have this pass?

Reply:
Can we adjust the tolerances instead?  3e-6 seems like a reasonable tolerance.

Reply:
I think it's better to disable TF32, as if someone modifies this test or adds new test methods, they won't have to worry about setting the tolerances to pass with Ampere.